modules/zip/src/index.ts

```diff
@@ -8,7 +8,8 @@ export {TarBuilder} from './tar-builder';
 export {
   parseZipCDFileHeader,
   makeZipCDHeaderIterator,
-  signature as cdSignature
+  signature as cdSignature,
```

Comment:
Can we export is as cdSignature so that we don't need to rename it?
For constants we often use `CAPITAL_CASE` i.e. `CD_SIGNATURE`